pkg/harvester/models/devices.harvesterhci.io.usbdevice.js

```diff
@@ -0,0 +1,159 @@
+import SteveModel from '@shell/plugins/steve/steve-class';
+import { escapeHtml } from '@shell/utils/string';
+import { HCI } from '../types';
+
+const STATUS_DISPLAY = {
+  enabled: {
+    displayKey: 'generic.enabled',
+    color:      'bg-success'
+  },
+  pending: {
+    displayKey: 'generic.inProgress',
+    color:      'bg-info'
+  },
+  disabled: {
+    displayKey: 'generic.disabled',
+    color:      'bg-warning'
+  },
+  error: {
+    displayKey: 'generic.disabled',
+    color:      'bg-warning'
+  }
+};
+
+/**
+ * Class representing USB Device resource.
+ * @extends SteveModal
+ */
+export default class USBDevice extends SteveModel {
+  get _availableActions() {
+    const out = super._availableActions;
+
+    out.push(
+      {
+        action:     'enablePassthroughBulk',
+        enabled:    !this.passthroughClaim && !this.status.enabled,
+        icon:       'icon icon-fw icon-dot',
+        label:      'Enable Passthrough',
+        bulkable:   true,
+        bulkAction: 'enablePassthroughBulk'
```

Comment:
Let's add `weight` for two bulk action to prevent button switch when sorting the columns.

See https://github.com/torchiaf/dashboard/blob/c3a6c31e4113d67bf5b8550ee19fb98c56f11b9f/shell/components/SortableTable/selection.js#L108

Reply:
Good catch!